mitosheet/src/mito/utils/keyboardShortcuts.tsx

```diff
@@ -286,15 +286,52 @@ export const keyboardShortcuts: KeyboardShorcut[] = [
     }
 ]
 
+/**
+ * Used to determine if the user is on a mac or windows.
+ * @returns 'mac' if the user is on a mac, 'windows' otherwise.
+ */
+export const getOperatingSystem = () => {
+    return window.navigator.userAgent.toUpperCase().includes('MAC')
+        ? 'mac'
+        : 'windows';
+}
+
+/**
+ * @param action - The action to get the keyboard shortcut for.
+ * @returns A string describing the keyboard shortcut for the given action.
+ */
+export const getKeyboardShortcutString = (action: Action) => {
```

Comment:
Wait, so we're notably not using the keyboard shortcuts defined in the action themselves. This is resulting in a O(actions * shortcuts) work when repeatedly calling this function. 

I know its a bit of duplicated data, but let's go through and just define the keyboard shortcuts in each action that has them. Then, let's just skip the .find step here, and use those.